.gitignore

```diff
@@ -50,6 +50,14 @@ obj/
 *.idb
 *.opendb
 
+# ======================= #
+# MSBuild4Unity generated #
```

Comment:
small nit, but there's already an `MSBuildForUnity generated` section at the bottom of the file